docs/data/date-pickers/custom-field/custom-field.md

```diff
@@ -132,6 +132,13 @@ you can replace the field with an `Autocomplete` listing those dates:
 
 {{"demo": "PickerWithAutocompleteField.js", "defaultCodeOpen": false}}
 
+### Using a masked `TextField`
+
+If you want to use a simple mask approach for the field editing instead of the built-in logic,
+you can replace the default field with a Text Field using a masked input value built with the [rifm](https://www.npmjs.com/package/rifm) package.
```

Comment:
```suggestion
If you want to use a simple mask approach for field editing instead of the built-in logic, you can replace the default field with a Text Field using a masked input value built with the [rifm](https://www.npmjs.com/package/rifm) package.
```

Reply:
The line break is there just for readability.
It does not impact DOM.
I would not change anything here. 🤔 
One idea is to maybe link to GH instead of npmjs, WDYT?

Reply:
For readability of the md file (just to be sure it's clear)

Reply:
> One idea is to maybe link to GH instead of npmjs, WDYT?

Updated

Reply:
💯

Reply:
IMO it doesn't feel more readable with the line break at all—it just looks random, like it was added by mistake. If the sentence is difficult to read because of the length, I would rather see it revised to be shorter.